### ISSUE TITLE
Render MDX in app?

### ISSUE THREAD

**Issue by reubenjs** [onejs/one#227]
Following guide for "Setting up MDX for web" works fine for web rendered (as the name suggests) but I can't get the MDX to work as an app e.g. iOS. 

It errors when trying to getMDXBySlug , I think because the rendered .mdx is not available?

any tips to get this going? thanks.

```
[error] Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.

Check your code at [slug].tsx:25.
```

**Comment by natew** [comment]
Yes, we don't have mdx set up for native yet. It requires a custom native library you'd have to install, and that requires different plugins and setup process. It'd be nice to do eventually for sure. Going to move this to a discussion.
